UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.UnitTests/Services/EssWebhookServiceTests.cs

```diff
@@ -210,6 +287,40 @@ private EnterpriseEventCacheDataRequest GetCacheRequestData()
             };
         }
 
+        private EnterpriseEventCacheDataRequest GetCacheRequestS57Data()
+        {
+            BatchDetails linkBatchDetails = new BatchDetails()
+            {
+                Href = @"http://tempuri.org.uk/batch/7b4cdb10-ddfd-4ed6-b2be-d1543d8b7272"
+            };
+            BatchStatus linkBatchStatus = new BatchStatus()
+            {
+                Href = @"http://tempuri.org.uk/batch/7b4cdb10-ddfd-4ed6-b2be-d1543d8b7272/status"
+            };
+            Get linkGet = new Get()
+            {
+                Href = @"http://tempuri.org.uk/batch/7b4cdb10-ddfd-4ed6-b2be-d1543d8b7272/files/exchangeset123.zip",
+            };
+            CacheLinks links = new CacheLinks()
+            {
+                BatchDetails = linkBatchDetails,
+                BatchStatus = linkBatchStatus,
+                Get = linkGet
+            };
+            return new EnterpriseEventCacheDataRequest
+            {
+                Links = links,
+                BusinessUnit = "ADDS-S57",
+                Attributes = new List<Attribute> { new Attribute { Key= "Agency", Value= "DE" } ,
+                                                           new Attribute { Key= "CellName", Value= "DE416050" },
+                                                           new Attribute { Key= "EditionNumber", Value= "2" } ,
+                                                           new Attribute { Key= "UpdateNumber", Value= "0" },
+                                                           new Attribute { Key= "ProductCode", Value= "AVCS" }},
+                BatchId = "d6cd4d37-4d89-470d-9a33-82b3d7f54b6f",
+                BatchPublishedDate = DateTime.UtcNow
+            };
+        }
```

Comment:
Accept BuisnessUnit all details are same in GetCacheRequestS63Data() method can we change GetCacheRequestS63Data() to GetCacheRequestData(string exchangeSetStandard) and set BusinessUnit = exchangeSetStandard to avoid duplicate code and remove GetCacheRequestS57Data() method please